cob_bringup/robots/raw3-1.xml

```diff
@@ -40,6 +40,11 @@
 			<include file="$(find cob_bringup)/drivers/emstate_phidget.launch">
 				<arg name="robot" value="$(arg robot)"/>
 			</include>
+			<!-- Relayboard with sim:=True is necessary because the robot does not have a phidgets board for checking the emergency status -->
+			<include file="$(find cob_bringup)/drivers/relayboard.launch">
+				<arg name="robot" value="$(arg robot)"/>
+				<arg name="sim" value="True"/>
+			</include>
```

Comment:
well, in that case the `X_phidget.launch` files should be removed, as they are not used!
Please also remove the obsolete `X_phidget.yaml` files in `cob_hardware_config`.

---
nit-picking: `True` -> `true`
```
<include file="$(find cob_bringup)/drivers/relayboard.launch">
	<arg name="robot" value="$(arg robot)"/>
	<arg name="sim" value="true"/>
</include>
```